### PR TITLE
Added important note about GPO and added missing link for download.

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving issue #3420 with an important note.

### CODE REVIEW

**Comment by e0i** [comment]
@j0rt3g4 Please reference the issue ID. Thanks.

**Comment by e0i** [comment]
@j0rt3g4 Please confirm whether the PR relates to issue #3420

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by j0rt3g4** [comment]
@e0i  Updated.

**Comment by e0i** [comment]
@mapalko Would you be able to review and merge this PR? Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-policy.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ Sign-in a domain controller or management workstations with _Domain Admin_ equiv
 2.	In the navigation pane, expand the domain and expand the node that has your Active Directory domain name.  Right-click the **Domain Controllers** organizational unit and click **Link an existing GPO�**
 3.	In the **Select GPO** dialog box, select **Domain Controller Auto Certificate Enrollment** or the name of the domain controller certificate enrollment Group Policy object you previously created and click **OK**.
 
+>[!IMPORTANT]
+>If you don't find options in GPO, you have to load "[policy definition folders](https://support.microsoft.com/help/3087759/how-to-create-and-manage-the-central-store-for-group-policy-administra)".
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-policy.md]:
```suggestion
>If you don't find options in GPO, you have to load the [PolicyDefinitions folder](https://support.microsoft.com/help/3087759/how-to-create-and-manage-the-central-store-for-group-policy-administra)". 
```

**Comment by j0rt3g4** [comment]
@JohanFreelancer9  done, thank you

**Comment by e0i** [comment]
@mapalko Would you be able to review and merge this PR based on issue #3420 ?

Thank you.

**Pull request merged**